Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -1133,6 +1134,7 @@ wrongLogin=Fehler bei Authentifizierung
 year=Jahr
 years=Jahre
 yes=Ja
+youWillBeRedirected=Sie werden nach erfolgreichem Speichern weitergeleitet.
```

Comment:
This would be the fourth redirection message. Other three are `ldapgroupYouWillBeRedirected`, `projectYouWillBeRedirected` and `userYouWillBeRedirected`. Maybe you should add an prefix like `docket` or this messages should be unique used.

Reply:
There will even be a fifths redirect with the identical text in the upcoming pull request for the ruleset editing page by @oliver-stoehr . Since he doesn't have the neutral "youWillBeRedirected" message _without_ object type that I added here, we agreed to replace all object specific messages with this general one and remove the obsolete object specific messages after both PRs were merged.

Reply:
Then should @oliver-stoehr not add a fifth redirect message and change it this neutral message key. Afterwards this messages should be unified.